Sources/Log.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+#include <QtDebug>
+#include <QFile>
+#include <QTextStream>
+#include <QUuid>
+#include "KVAbort.h"
+#include <QDateTime> 
+
+void InformationToStream(const QString type, const QMessageLogContext &context, const QString &message, const QString &datetime);
+void KvMessageHandler(QtMsgType type, const QMessageLogContext &context, const QString &message);
+void InstallMessageHandler();
```

Comment:
Functions which are used only in `Log.cpp` should not be declared in the header.